### PR TITLE
Configuration form pass

### PR DESCRIPTION
**Pull request by Elkrival**
This pr adds a simple layout to the new configuration and edit configuration pages.

### CODE REVIEW

File: views/forms/ConfigForm/ConfigForm.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.ConfigFormActions {
+  position: fixed;
```
Comment (JackVCurtis) [inline views/forms/ConfigForm/ConfigForm.css:2]: Usually this is a red flag for me, but Material is weird. Is `position: fixed` recommended somewhere? What problem is this resolving?

File: views/forms/TextInput/index.jsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ const TextInput = (props) => {
       type={props.type}
       placeholder={props.placeholder}
       sx={props.sx}
+      disabled={props.disabled}
```
Comment (JackVCurtis) [inline views/forms/TextInput/index.jsx:23]: If we're passing through `props` can we use `...props` instead?

File: views/pages/EditConfigPage.jsx [hunk 3]
```diff
@@ -66,20 +64,29 @@ const EditConfigPage = () => {
...
-      fields={fields}
-      friendsList={friendsList}
-      gridState={gridState}
-      onAddNewField={addNewField}
-      onCopy={onCopy}
-      onSubmit={handleSubmit(handleFormData)}
-      onRemove={removeField}
-    />
+    <Box sx={{ p: '30px' }}>
+      <Typography variant="h6">Edit Configuration</Typography>
```
Comment (JackVCurtis) [inline views/pages/EditConfigPage.jsx:78]: This is just an aside but WTF is wrong with an `h6` tag `@mui`?

File: views/forms/ConfigForm/ConfigForm.css (hunk 1 above)
Comment (Elkrival) [reply views/forms/ConfigForm/ConfigForm.css:2]: this keeps the add a configuration card visible on screen, there can be up anywhere from 1-400+ cards on screen. By having it visible as a user scrolls they can add more cards as they need

File: views/forms/TextInput/index.jsx (hunk 2 above)
Comment (Elkrival) [reply views/forms/TextInput/index.jsx:23]: yeah, I think we want to be more deliberate with these props if we spread the props we have a chance of adding a prop that may not be present in the component

File: views/pages/EditConfigPage.jsx (hunk 3 above)
Comment (Elkrival) [reply views/pages/EditConfigPage.jsx:78]: haha, yeah it's weird with the variants

File: views/forms/ConfigForm/ConfigForm.css (hunk 1 above)
Comment (JackVCurtis) [reply views/forms/ConfigForm/ConfigForm.css:2]: So it is fixed relative to the whole page, no weird `position: relative` stuff going on? Okay, works for me.

**Review by JackVCurtis — APPROVED** [review]
👍

**Pull request merged**